crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java

```diff
@@ -111,7 +122,7 @@ public Response getSearchesForEntity(@PathParam("entity") String entity,
         }
         CallStatus st=new FindCommand(freq.getEntityVersion().getEntity(),
                                       freq.getEntityVersion().getVersion(),
-                                      freq.toJson().toString()).run();
+                                      freq.toJson().toString(), metrics).run();
```

Comment:
Actually, I had an idea about this I just commented on the other PR. We have the option to start tracking in the REST layer (here), and then pass the context along.

Another alternative is to go back to doing most tracking in the REST layer, but for bulk requests just pass along the RequestMetrics so it can track the individual requests itself. You'll still have needed the RequestMetrics interface in core to accomplish that.